process/src/GameObject.ts

```diff
@@ -534,6 +545,14 @@ class GameObject {
       result.blocksWalking = true;
     }
 
+    // The game supports tapout data both as a separate line in the object file, or as a comment in the object name.
+    // Tapout data in comments is not currently being utilised, and not supported here, however will still display 
+    // in its raw form with other object comments.
+    if (this.data.tapoutTrigger && this.data.tapoutTrigger.length > 1) {
```

Comment:
Nit: Can you also detail the necessity to check length? This seems to break from the pattern of other checks.